.travis.yml

```diff
@@ -13,7 +13,7 @@ addons:
 before_install:
   - if test "$TRAVIS_OS_NAME" == "osx"; then brew update; brew install icu4c; fi
 env:
-  - KOREBUILD_DNU_RESTORE_CORECLR=true MONO_THREADS_PER_CPU=2000 MONO_MANAGED_WATCHER=disabled
+  - KOREBUILD_DNU_RESTORE_CORECLR=true MONO_THREADS_PER_CPU=2000 MONO_MANAGED_WATCHER=disabled KOREBUILD_TEST_DNXCORE=true
```

Comment:
Looks like the `NO_PARALLEL_TEST_PROJECTS` environment variable needs to include `Microsoft.AspNet.Mvc.FunctionalTests`.